# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../fam/fam-2.7.0-limits.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN fam-2.7.0-orig/include/fam.h fam-2.7.0/include/fam.h
--- fam-2.7.0-orig/include/fam.h	2003-01-20 05:22:44.000000000 +0100
+++ fam-2.7.0/include/fam.h	2010-08-29 14:20:45.000000000 +0200
@@ -41,7 +41,7 @@
 *****************************************************************************/
 
 /* For NAME_MAX - maximum # of chars in a filename */
-#include "limits.h"
+#include <limits.h>
 
 
 
